packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -253,6 +253,8 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
       withLoading
         ? Template.asString([
             new DynamicFileSystem().generate(),
+            `if(!globalThis.__remote_scope__) globalThis.__remote_scope__ = ${RuntimeGlobals.require}.federation`,
```

Comment:
Backward compatibility and makes older parts of plugin "just work" under new structure